handlers/bootstrap.go

```diff
@@ -139,8 +140,11 @@ func buildContainerConfig(containerCmd []string, machine *client.Machine, imgRep
 			continue
 		}
 		labelPair := key + "=" + label
-		envVars = append(envVars, labelPair)
+		labelVars = append(labelVars, labelPair)
 	}
+	labelVarsString := strings.Join(labelVars, "&")
+	labelVarsString = "CATTLE_HOST_LABELS=" + labelVarsString
+	envVars = append(envVars, labelVarsString)
```

Comment:
What if machine.Labels is empty? Seems to me that we would not want to add "CATTLE_HOST_LABELS=" (without a value) to the run command.